docs/build-docs-locally.rst

```diff
@@ -0,0 +1,40 @@
+Build Docs Locally
+==================
+
+Few steps are required to build the docs locally
+with this extension.
+
+.. note::
+
+    To make this extension work locally, it is important to have
+    `local instance of Read the Docs running`_.
```

Comment:
I think we need to make it clear that this isn't a requirement of using the extension. The extension gracefully degrades when it can't get search results or query an API. We want project maintainers to have confidence adding this extension to their project. It should add functionality when deployed on Read the Docs and simply do nothing otherwise.

Ideally maybe there's a way to hook into its functionality to get search results from another source but I don't think we need that quite yet.